certificate/certificate.go

```diff
@@ -359,15 +367,21 @@ func getCertExtensions(cert *x509.Certificate) Extensions {
 	crld = append(crld, cert.CRLDistributionPoints...)
 	pnames := make([]string, 0)
 	pnames = append(pnames, cert.PermittedDNSDomains...)
+	constraints, _ := certconstraints.Get(cert)
```

Comment:
Yes. The only reason it can fail is if somehow an invalid/corrupted certificate makes it here, and afaict there's checks when the certificate is loaded to prevent that.